src/pages/proposal-details/index.tsx

```diff
@@ -0,0 +1,328 @@
+	<ValidatorName index={rowIndex} moniker={data.moniker} identity={data.identity} />
+);
+
+const AnswerCell: FunctionComponent<CellRendererProps<any, ParsedValidator>> = observer(({ data, rowIndex }) => {
+	console.log('========\n', 'data', data, '\n========');
+	const { chainStore, queriesStore } = useStore();
+
+	const votingData = queriesStore
+		.get(chainStore.current.chainId)
+		.rebus.queryValidatorVoting.getVotingRecord(data.proposalId, data.address);
+
+	console.log('========\n', 'votingData', votingData.response, '\n========');
+	return <div>data.status</div>;
+});
+
+const ValidatorTable: FunctionComponent<ProposalDetailParam> = ({ proposalId }) => {
+	const { inProgress, validatorList } = useValidators();
+
+	const validatorData = useMemo(() => {
+		const activeValidators = validatorList?.filter((item: any) => item.status === 3);
+
+		console.log('========\n', 'activeValidators', activeValidators, '\n========');
+
+		return activeValidators && activeValidators.length
+			? activeValidators.map(item => {
+					const parsedItem: ParsedValidator = {
+						moniker: item.description.moniker,
+						identity: item.description.identity,
+						status: item.status,
+						address: item.operator_address,
+						proposalId,
+					};
+
+					return parsedItem;
+			  })
+			: EMPTY_TABLE_DATA;
+	}, [proposalId, validatorList]);
+
+	const columnDefs: ColumnDef<ParsedValidator>[] = [
+		{
+			property: 'moniker',
+			header: 'Validator',
+			CellRenderer: ValidatorCell,
+			width: 4,
+			headerAlign: 'center',
+			canSort: true,
+			hideHeaderMobile: true,
+		},
+		{
+			property: 'status',
+			header: 'Answer',
+			CellRenderer: AnswerCell,
+			width: 2,
+			align: 'center',
+			headerAlign: 'center',
+		},
+	];
+
+	const noDataFound = <div className="no_data_table"> No data found </div>;
+	const loader = <CircularProgress />;
+	return (
+		<DataTable
+			className={'h-full'}
+			columnDefs={columnDefs}
+			data={validatorData}
+			loading={inProgress}
+			loader={loader}
+			minWidth="1270px"
+			noData={noDataFound}
+			tableRowClassName={'my-table-row-class'}
+		/>
+	);
+};
+
+const ProposalDetailPage: FunctionComponent = observer(() => {
+	const { proposalId } = useParams<ProposalDetailParam>();
+	const { chainStore, queriesStore } = useStore();
+	const queries = queriesStore.get(chainStore.current.chainId);
+	const proposal = queries.cosmos.queryGovernance.getProposal(proposalId);
+
+	const formatLinks = (txt: string) => {
+		return txt.split(' ').map(part =>
+			URL_REGEX.test(part) ? (
+				<a className="text-primary-50" target="_blank" href={part} rel="noreferrer">
+					{part}{' '}
+				</a>
+			) : (
+				part + ' '
+			)
+		);
+	};
+
+	const descriptionFormatter = (text: string) => {
+		return text.split('\\n\\n').map((str, index) => (
+			<p className="my-1.5" key={index}>
+				{formatLinks(str)}
+			</p>
+		));
+	};
+
+	const PieChart = () => {
+		// const data = [
```

Comment:
Let's remove this code if its not being used

Reply:
sorry still testing out charts updated the PR to be WIP